eras/alonzo/impl/CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## 1.9.0.0
 
+* Add `LedgerTxInfo` to `Cardano.Ledger.Alonzo.Plutus.Context`
+* `EraPlutusTxInfo` class:
+  * replace params in `toPlutusTxInfo` signature with `LedgerTxInfo`
+  * replace `toPlutusScriptContext` with `toPlutusArgs`
+* `EraPlutusContext` class:
+  * remove `mkPlutusScriptContext`, `mkPlutusLanguageContext`
+  * add `mkPlutusWithContext`
+* add `toPlutusWithContext` to `Cardano.Ledger.Alonzo.Plutus.Context`
+* Move `PlutusScriptContext` type family to core
+* Add `ContextError` inhabitant to `TransactionScriptFailure`
+* Change return type of `evalTxExUnits` and `evalTxExUnitsWithLogs` to not be `Either`s
+* Add `lookupPlutusScript` and `eraLanguages` to Alonzo `Scripts`
+* Deprecate `lookupPlutusScript` in Alonzo `Evaluate`
```

Comment:
Thank you for adding these. Totally forgot about changelogs in CIP69 work

Reply:
Happy to contribute!